### PR TITLE
Support F609 different header

### PR DESCRIPTION
**Pull request by ChrisG661**
I can decode with type-0 and  type-0 creates a header that almost match the original, though encoding with type-0 makes the payload different. I am using these arguments to encode the config:
`--signature F609 --payload-type 2 --include-unencrypted-length --version 1 decoded.xml encoded.bin`
Using `payload-type 2` works and the payload are perfect, though it 128 first bytes which are not required.
I can provide the original header if required.
Right now I am not sure if any other model have the same header as the F609.
I modified the payload type check to skip if signature is `F609`

### CODE REVIEW

**Comment by mkst** [comment]
Hi @ChrisG661, firstly thanks for taking an interest in this project!

I'm not super-keen on having a specific router model hardcoded (i.e. if "F609" do this/that), it would be better as (yet another) command-line option.

What could then be done would be to have some lookup table mapping signatures to configuration options, such that if you added a signature of "F600W" (or whatever) it would enable e.g. "payload-type 0".

Obviously that is more work than an if statement, but is a nicer approach - as if someone were to come along with a similar request, the lazy option now would be to just add an `and signature != "something_else"` to your modification.. which is not a nice path to go down.

TL;DR: I'll allow this, but if I find the time, I'll look into making this a command-line option and removing this specific if statement... the next step would be the LUT, but that's more work.
